### PR TITLE
Posts: hide comment moderation tools on the Reader

### PR DESCRIPTION
**Pull request by rodrigoi**
Adds new properties to show the comment filters and moderation tools only on Posts.

### CODE REVIEW

**Comment by blowery** [comment]
Do we _want_ to hide the moderation tools in the reader if you have the ability to moderate the comments? Seems like a natural use for an author viewing their own stuff in the Reader.

cc @fraying

**Comment by rodrigoi** [comment]
We should include the filters too, if the user can `moderate_comments`.

**Comment by fraying** [comment]
I'd be totally into showing the mod tools if we can limit them just to the owner of the site. We currently show an Edit link in the post tools if the user is the owner. This would match that.

File: client/blocks/comments/post-comment-list.jsx [hunk 1]
```diff
@@ -88,6 +88,7 @@ class PostCommentList extends React.Component {
 			commentsTree={ this.props.commentsTree }
 			commentId={ commentId }
 			key={ commentId }
+			showModerationTools={ this.props.showModerationTools }
```
Comment (samouri) [inline client/blocks/comments/post-comment-list.jsx]: this is a better place to put the `config.isEnabled( 'moderation-tools-in-posts ')`

File: client/my-sites/posts/post.jsx [hunk 2]
```diff
@@ -333,6 +333,8 @@ const Post = React.createClass( {
 					<Comments
 						showCommentCount={ false }
 						post={ this.props.post }
+						showFilters
+						showModerationTools
```
Comment (samouri) [inline client/my-sites/posts/post.jsx]: Would this be a good place to put: `showModerationTools={ config.isEnabled( ... ) }`?

**Comment by samouri** [comment]
Code needs a rebase. testing out functionality now

**Comment by rodrigoi** [comment]
Depends on https://github.com/Automattic/wp-calypso/pull/12551.

**Comment by fraying** [comment]
As discussed over in #12551, totally cool to omit the new comment mod tools from Reader for now.

File: client/my-sites/posts/post.jsx (hunk 2 above)
Comment (roundhill) [inline client/my-sites/posts/post.jsx]: Seems like a good idea, unless I'm missing something 🤔

**Comment by alisterscott** [comment]
I'm seeing the original issue (#12540) where no comments are shown for all reader readers (fixed in #12551), so am I right in saying that the #12551 needs to be merged _before_ this one so we don't see that same issue again?

**Comment by rodrigoi** [comment]
@samouri @fraying @roundhill this is ready for another review. Tools and filters will be hidden from the _Reader_ for now. Feature flag has been refactored and there's one arrow function I'll remove later as part of another refactor/reorg of this components.

**Comment by fraying** [comment]
I checked it and the comments in Reader look right! ("Right" being no mod tools, and comments present.) Thanks, @rodrigoi!

**Pull request merged**